tests/numpy/test_arithmetic.py

```diff
@@ -144,9 +144,78 @@ def test_inner_outer(nps_app_inst):
     assert np.allclose(np.inner(A, B), nps.inner(nps_A, nps_B).get())
     assert np.allclose(np.outer(A, B), nps.outer(nps_A, nps_B).get())
 
+# TODO (hme): Add broadcast tests.
 
-# TODO(hme): Add broadcast tests.
+def test_tensordot_shape_error(nps_app_inst):
+    import numpy as np
+    from nums.core.array.utils import is_array_like
+    from nums import numpy as nps
+    import pytest
```

Comment:
Can you move numpy and pytest imports to the top of the file?

Reply:
I've also went ahead and moved all the nums imports to the top as well. It should look much neater and consistent now.